### ISSUE TITLE
GPS module should not be mandatory

### ISSUE THREAD

**Issue by KenLWalker** [Bowerbird/bowerbird-web#39]
In the Observation upload, the GPS module is not shown as being "Required" however, if you try to Save the record without providing a GPS then you get blocked and a Red banner warns you to complete the GPS coordinates.  Please change it to a non-mandatory field.

![2013-03-19_1428](https://f.cloud.github.com/assets/3622867/274005/5ea7a72a-9045-11e2-9671-f679d4a829a2.png)

**Comment by frankradocaj** [comment]
I don't think it would be a good idea to make location non-mandatory. The entire system hinges on sightings having a location. If you don't drag the pin, or enter an address, currently you will see the following error message in red:

"Please enter a location. Either drag and drop the pin or enter an exact coordinate (click the options button)."

I don't see any reference to "GPS". Are you referring to the message above?

**Comment by KenLWalker** [comment]
Thanks Frank.

Currently, we are scanning and uploading a series of fungal slides donated to us many years ago (see below).  None of these slides have precise locality data other than we know they were all taken in Victoria.

When we load these images we are setting the location to the Melbourne CBD with a note in comments that we do not know the localities for these images.

I guess this is what I was thinking about when requesting a non-mandatory GPS field.

Perhaps there could be a checkbox – Locality Unknown.

Just a thought.  I do not mind which way this goes.

Cheers,

Ken

[cid:image001.png@01CE3F59.F31238E0]
